components/coding/GoalsSectionComponent.tsx

```diff
@@ -80,14 +80,12 @@ export default function GoalsSection({
         })}
       </div>
     </ExpandableContainer>
-  ) : userGoals.length <= 0 &&
-    componentUsageType === GoalsComponentSection.PROFILE ? (
+  ) : userGoals.length <= 0 && inProfile === true ? (
```

Comment:
tip: when referencing a boolean, the default is always true so you can just write "inProfile" - does the same thing as your code above